package.py

```diff
@@ -0,0 +1,25 @@
+name = "studiolibrary"
+
+version = "2.16.0"
+
+authors = [
+    "krathjen",
+    "Jeremy Andriambolisoa",
+]
+
+description = \
+    """
+    Manage poses, animation and digital assets in Autodesk Maya.
+    """
+
+requires = [
+    "python-3+",
+    "maya-2025"
+]
+
+uuid = "krathjen.studiolibrary"
+
+build_command = 'python {root}/build.py {install}'
+
+def commands():
+    env.PYTHONPATH.append("{root}/python/studiolibrary/src")
```

Comment:
Instead of copying the full content of the studiolibrary repository to the packages folder, you should probably only move the content of `src` to `{root}/python/studiolibrary` and not keep all the files in the released package hierarchy.